src/components/Timeline/Event/Events.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import ErrorMessage from 'components/elements/ErrorMessage'
+import LoadingMessage from 'components/elements/LoadingMessage'
+import Highlight from 'components/elements/Highlight'
+import EventCard from 'components/Timeline/Event/EventCard'
+import BootstrapPlaybook from 'components/Timeline/Playbook/BootstrapPlaybook'
+import * as eventTypeActions from 'actions/eventTypeActions'
+
+export const Events = ({events, ticketId, incidentId}) => <div>
+  {Array.from(events)
+    .map(event =>
+      <ConnectedEvent
+        key={event.id}
+        event={event}
+        ticketId={ticketId}
+      />)
+  }
+</div>
+
+Events.propTypes = {
+  events: PropTypes.array.isRequired,
+  ticketId: PropTypes.string.isRequired,
+  incidentId: PropTypes.number
+}
+
+const eventHasValidDisplayText = (event) =>
+  event && event.data && event.data.DisplayText
+
+export const Event = ({
+  eventTypeIsFetching,
+  eventTypeIsError,
+  event,
+  ticketId
+}) => eventTypeIsFetching && !eventHasValidDisplayText(event)
```

Comment:
Move the negation into `eventHasValidDisplayText` and rename to `eventHasInvalidDisplayText`.